packages/bits/src/lib/select-v2/combobox-v2/combobox-v2.component.ts

```diff
@@ -122,6 +122,8 @@ export class ComboboxV2Component extends BaseSelectV2 implements AfterContentIni
         // we check "selectedOptions" to be set per "value" again in "handleValueChange"
         this.optionsChanged().subscribe(() => {
             this.filterItems(this.inputValue.toString());
+            // We need the active option to always stay visible during the options filtering.
+            this.optionKeyControlService.scrollToOption({block: "start", behavior: "smooth"});
```

Comment:
I know y'all are pressed for time, but do you have time to write a unit test that would be red before this fix and green after the fix?

Reply:
added e2e test to track the regression